otterdog/eclipse-tractusx.jsonnet

```diff
@@ -228,6 +228,12 @@ orgs.newOrg('eclipse-tractusx') {
       workflows+: {
         default_workflow_permissions: "write",
       },
+      branch_protection_rules: [
+        orgs.newBranchProtectionRule('main') {
+        dismisses_stale_reviews: true,
+        required_approving_review_count: 2,
```

Comment:
As this PR is not introduceing branch protection and required review count to the GH Org but to the eclipse-tractusx/eclipse-tractusx.github.io repo only, I would like to require 2 reviewers. For this repository enough stakeholders should be available doing reviews. 

Maybe we should also think about to introcude `.github/CODEOWNERS` (for eclipse-tractusx/eclipse-tractusx.github.io). Code owners are automatically assigned to new PRs:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Reply:
I think the idea of a `.github/CODEOWNERS` file would be pretty useful. This would also solve the problem of Issues and PRs not associated. We could divide it for every section (like KIT etc.).